addons/website_event_sale/static/tests/tours/website_event_sale_last_ticket.js

```diff
@@ -0,0 +1,85 @@
+        {
+            content: "Show available Tickets",
+            trigger: '.btn-primary:contains("Register")',
+        },
+        {
+            content: "Select 2 units of `VIP` ticket type",
+            extra_trigger: '#wrap:not(:has(a[href*="/event"]:contains("Last ticket test")))',
+            trigger: 'select:eq(0)',
+            run: 'text 2',
+        },
+        {
+            content: "Click on `Order Now` button",
+            extra_trigger: 'select:eq(0):has(option:contains(2):propSelected)',
+            trigger: '.a-submit:contains("Register")',
+        },
+        {
+            content: "Fill attendees details",
+            trigger: 'form[id="attendee_registration"] .btn:contains("Continue")',
+            run: function () {
+                $("input[name='1-name']").val("Att1");
+                $("input[name='1-phone']").val("111 111");
+                $("input[name='1-email']").val("att1@example.com");
+                $("input[name='2-name']").val("Att2");
+                $("input[name='2-phone']").val("222 222");
+                $("input[name='2-email']").val("att2@example.com");
+            },
+        },
+        {
+            content: "Validate attendees details",
+            extra_trigger: "input[name='1-name'], input[name='2-name']",
+            trigger: 'button:contains("Continue")',
+        },
+        {
+            content: "Fill address",
+            trigger: 'form.checkout_autoformat',
+            run: function () {
+                $("input[name='name']").val("test1");
+                $("input[name='email']").val("test@example.com");
+                $("input[name='phone']").val("111 111");
+                $("input[name='street']").val("street test 1");
+                $("input[name='city']").val("testCity");
+                $("input[name='zip']").val("123");
+                $('#country_id option:eq(1)').attr('selected', true);
+            },
+        },
+        {
+            content: "Validate address",
+            trigger: '.btn-primary:contains("Next")',
+        },
+        {
+            // if the seats_available checking logic is not correct,
+            // the shopping cart will be cleared when selling the last ticket
+            // the tour test will be failed here
+            content: "Select `Wire Transfer` payment method",
+            trigger: '#payment_method label:contains("Wire Transfer")',
+        },
+        {
+            content: "Pay",
+            //Either there are multiple payment methods, and one is checked, either there is only one, and therefore there are no radio inputs
+            extra_trigger: '#payment_method label:contains("Wire Transfer") input:checked,#payment_method:not(:has("input:radio:visible"))',
+            trigger: 'button[id="o_payment_form_pay"]:visible',
+        },
+        {
+            content: "Last step",
+            trigger: '.oe_website_sale:contains("Please make a payment to:")',
+            timeout: 30000,
+        }
```

Comment:
I see that this part is copied from a tour in website_sale.

Maybe you can add a comment saying that it's copied from there?
Just because that huge timeout looks strange.
(Also, it seems like they have an additional step that checks that you correctly land on the "contactus" page).

Reply:
I was talking about this part:
https://github.com/odoo/odoo/blob/14.0/addons/website_sale/static/tests/tours/website_sale_buy.js#L88

And the subsequent step that checks that we landed on the contactus page.

But I see it's not done the same way in:
https://github.com/odoo/odoo/blob/14.0/addons/website_event_sale/static/tests/tours/website_event_sale.js#L92

So maybe it's fine after all? A bit confusing ^^